lib/collector.go

```diff
@@ -68,6 +68,24 @@ var (
 		"owner_crashes",
 		"worker_crashes",
 		"closes"}
+
+	exposedRequestTimes = []string{
```

Comment:
The `exposedRequestTimes` and `exposedPercentiles` variables aren't used anywhere - or did I miss something?

Reply:
Good catch. I have removed these variables.